shared/common-adapters/dumb.desktop.js

```diff
@@ -40,6 +41,20 @@ const checkboxMap: DumbComponentMap<Checkbox> = {
   },
 }
 
+const iconMap: DumbComponentMap<Icon> = {
```

Comment:
Ah I understand now, thanks. After discussing with Marco, he came up with an interesting idea of tiling multiple views per screenshot. I'm pretty convinced that'll work if we need to speed things up.